layout/container.ftd

```diff
@@ -13,10 +13,10 @@ optional ftd.resizing height:
 ds.gap margin-vertical: $ds.spaces.vertical-gap.small
 ds.gap margin-horizontal: $ds.spaces.vertical-gap.small
 integer border: $ds.borders.uniform.none
-integer border-top: $ds.borders.none
-integer border-bottom: $ds.borders.none
-integer border-left: $ds.borders.none
-integer border-right: $ds.borders.none
+optional integer border-top: $ds.borders.none
+optional integer border-bottom: $ds.borders.none
+optional integer border-left: $ds.borders.none
+optional integer border-right: $ds.borders.none
```

Comment:
If you specify a default value, you do not need optional!

@Arpita-Jaiswal / @harshdoesdev lets add a lint / warning for this (and lets add --strict mode soon, so our code does not contain this mistake).

Reply:
@MeenuKumari28, remove the part after the `:` when you are making something optional.